src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Updated [Http Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/http.md).
```

Comment:
link should point to database.md

Reply:
Should it?
The database attributes didn't change, only the Http ones.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/database.md#:~:text=Warning%20Existing%20Database,October%201%2C%202023)

Reply:
sorry, you're right. been looking at both docs this week. i'll get this corrected.